package.json

```diff
@@ -34,7 +34,10 @@
       "!**/node_modules/**",
       "!**/babel.config.js",
       "!**/jest.setup.js"
-    ]
+    ],
+    "moduleNameMapper": {
+      "^parse/react-native$": "parse/node"
```

Comment:
oh this is so cool, noice @nisargap !

Reply:
Thanks! I was originally trying to override the file, then I thought I could just map the module somehow lol